internal/cluster/cluster_service.go

```diff
@@ -139,7 +147,7 @@ func (s *Service) handleMembersUpdated(conn *Connection, version int32, memberIn
 
 func (s *Service) sendMemberListViewRequest(ctx context.Context, conn *Connection) error {
 	request := codec.EncodeClientAddClusterViewListenerRequest()
-	inv := s.invocationFactory.NewConnectionBoundInvocation(request, -1, nil, conn, func(response *proto.ClientMessage) {
+	inv := s.invocationFactory.NewConnectionBoundInvocation(request, -1, "", conn, func(response *proto.ClientMessage) {
```

Comment:
It feels like this `NewConnectionBoundInvocation` should not take `partition id` and `address` as a parameter in the first place. 
It is not legal to set those to a connection bound invocation.

Reply:
Updated at: https://github.com/hazelcast/hazelcast-go-client/pull/579/commits/5801b4149d02b4f7937c5e3d37095d3eda39c742